hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/WritableAccountStoreTest.java

```diff
@@ -93,31 +91,31 @@ void getForModifyLooksForAlias() {
     void getForModifyReturnEmptyIfAliasNotPresent() {
         writableStore.put(account);
 
-        final var maybeReadAccount = writableStore.getForModify(alias);
+        final var readaccount = writableStore.getForModify(alias);
 
-        assertFalse(maybeReadAccount.isPresent());
-        assertEquals(0, writableStore.sizeOfAliasesState());
+        assertThat(readaccount).isNull();
+        assertThat(writableStore.sizeOfAliasesState()).isZero();
     }
 
     @Test
     void putsAccountChangesToStateInModifications() {
-        assertFalse(writableStore.get(id).isPresent());
+        assertFalse(writableStore.get(id) != null);
```

Comment:
The other assertions in this test are Assertj now, so this one probably should be too

Reply:
Fixed. Thanks